nvflare/app_opt/xgboost/tree_based/executor.py

```diff
@@ -279,6 +279,16 @@ def train(
 
         self.local_model = bst.save_raw("json")
 
+        # remove the sum_hessian from local_model for privacy
+        if "learner" in self.local_model and "gradient_booster" in self.local_model["learner"]:
+            gradient_booster = self.local_model["learner"]["gradient_booster"]
+            if "model" in gradient_booster and "trees" in gradient_booster["model"]:
+                trees = gradient_booster["model"]["trees"]
+                for tree in trees:
+                    if "sum_hessian" in tree:
+                        # Hide sum_hessian with all-1
+                        tree["sum_hessian"] = [1.0] * len(tree["sum_hessian"])
```

Comment:
Can we not completely remove sum_hessian and just recreate it if needed by the xgboost code?

Reply:
it won't pass the xgb check for model loading, so we would need to add it back rightaway when it arrives client, which does not make any difference

Reply:
right but still useless to communicate it....